css/style.css

```diff
@@ -16,4 +15,125 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+header {
```

Comment:
instead of using the tag names in your CSS, it is better to create some classes and use those classes because when you have a bigger website, if you write your CSS in this way it will affect all of the tags on different pages.